WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController+Activity.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+// MARK: - SearchableActivity Conformance
+extension DashboardViewController: SearchableActivityConvertable {
+    var activityType: String {
+        return WooActivityType.dashboard.rawValue
+    }
+
+    var activityTitle: String {
+        return NSLocalizedString("My Store", comment: "Title of the 'My Store' tab - used for spotlight indexing on iOS.")
+    }
+
+    var activityDescription: String? {
+        return NSLocalizedString("See at a glance which products are winning.",
+                                 comment: "Description of the 'My Store' screen - used for spotlight indexing on iOS.")
+    }
+
+    var activityKeywords: Set<String>? {
+        let keyWordString = NSLocalizedString("woocommerce, my store, today, this week, this month, this year," +
+                                              "orders, visitors, conversion, top conversion, items sold",
+                                              comment: "This is a comma separated list of keywords used for spotlight indexing of the 'Dashboard' tab.")
+        let keywordArray = keyWordString.arrayOfTags()
+
+        guard !keywordArray.isEmpty else {
+            return nil
+        }
+
+        return Set(keywordArray)
+    }
+}
```

Comment:
💯 to extracting SearchableActivity conformance to a different file for Dashboard, Orders, Products, ...